harvard-cardiff-university.csl

```diff
@@ -84,11 +84,11 @@
   </macro>
   <macro name="title">
     <choose>
-      <if type="bill book graphic legal_case legislation motion_picture report song thesis" match="any">
-        <text variable="title" font-style="italic"/>
+      <if type="bill book graphic legal_case legislation motion_picture report song thesis patent" match="any">
+        <text variable="title" font-style="italic" prefix=" " suffix="."/>
```

Comment:
The prefix/suffix for the title of all other elements is coded in the bibliography section. Please remove them here and add them to the patent there. This probably works for other item types because Zotero/Mendeley are pretty good at preventing duplicate punctuation or spaces, but we try not to rely on that too much.